lib/apiservers/portlayer/restapi/handlers/containers_handlers.go

```diff
@@ -360,6 +361,13 @@ func (handler *ContainersHandlersImpl) GetContainerLogsHandler(params containers
 		return containers.NewGetContainerLogsInternalServerError()
 	}
 
+	// containers with PluginVersion > 0 will use updated output logging on the backend
+	// containers with ExecConfig.Version = nil are assumed to be PluginVersion 0
+	if container.ExecConfig.Version != nil && container.ExecConfig.Version.PluginVersion > 0 {
```

Comment:
I suggest checking the version explicitly as we can introduce another change later and that chance could be incompatible with this. > 0 test will cause problems in that case.

I also argue having a version numbers means nothing to the reader so I strongly suggest defining some constants and using them as conditions, eg [obviously not tested, not sure even the syntax is right :)];

```
const (
        TheVersionThatILike = 1 << iota
        TheVersionNobodyLikes
        TheBestVersionEver
        TheBrokenOne
        TheBest
        YuggggeeeVersion
)

CurrentPluginVersion = TheBest
```

and then you can create a function called Satisfies which takes an int argument and call it like

```
if err :=version.Satisfies(TheVersionThatILike | YuggggeeeVersion); err !=nil {
...
} else if err := version.Satisfies(TheBrokenOne); err !=nil  {
...
}
```

which could be implemented like

```
func (v *Version) Satisfies(flags int) error {
	if flags&TheVersionThatILike != 0 && CurrentPluginVersion < TheVersionThatILike {
		return ErrNotSupported
	}
....
	return nil
}
```